sdk/core/Azure.Core/src/RequestContent.cs

```diff
@@ -322,5 +336,43 @@ public override Task WriteToAsync(Stream stream, CancellationToken cancellation)
                 return Task.CompletedTask;
             }
         }
+        private sealed class ModelReadWriteRequestContent<T> : RequestContent
+        {
+            private BinaryData? _data;
+            public BinaryData Data => _data ??= GetData();
+            private IPersistableModel<T> _model;
+            private ModelReaderWriterOptions _options;
+
+            public ModelReadWriteRequestContent(IPersistableModel<T> model, ModelReaderWriterOptions options)
+            {
+                _model = model;
+                _options = options;
+                ContentType = Data.MediaType ?? "application/json";
+            }
+            private BinaryData GetData()
+            {
+                BinaryData data = ModelReaderWriter.Write(_model, _options); // will call the write function in serialization file
```

Comment:
our ModelReaderWriter can write to a sequence of buffers. But this Write method (returning BinaryData) forces the sequence to be copied to one contiguous buffer. Can we avoid that? Can we keep a sequence of buffers in ModelReadWriteRequestContent?

Reply:
Hello @KrzysztofCwalina I only can find the api `public static BinaryData Write<T>(T model, ModelReaderWriterOptions? options = default) where T : IPersistableModel<T>` in `ModelReaderWriter.cs`. And I don't see a API which return a sequence of buffers. 
Is it a new API which will be released in later version?